spec/system/support_interface/see_individual_application_spec.rb

```diff
@@ -35,6 +35,9 @@ def and_there_are_applications_in_the_system
       :with_gcses,
       references_state: :feedback_requested,
     )
+
+    create(:application_choice, application_form: @completed_application, status: 'unsubmitted')
```

Comment:
why is this needed?

Reply:
I think we need it to create an instance of `application_choice` so that `application_form.application_choices.first` isn't `nil` so the mailer can call `reject_by_default_at` on it